app/api/v2/atbds.py

```diff
@@ -151,6 +154,28 @@ def create_atbd(
     return atbd
 
 
+@router.post("/atbds/rebuild-index")
+def rebuild_atbd_index(
+    principals: List[str] = Depends(get_active_user_principals),
+):
+    """Rebuild ATBD index from scratch"""
+    check_atbd_permissions(
+        principals=principals, action="rebuild_atbd_index", atbd=None
+    )
+    task_queue = get_task_queue()
+    task_queue.send_message(
+        MessageBody=base64.b64encode(
+            pickle.dumps(
+                {
+                    "task_type": "rebuild_atbd_index",
+                    "payload": {},
+                }
+            )
+        ).decode()
+    )
```

Comment:
Now that we have more than one task types, I think we should build a reusable abstraction to add tasks. But I'll do that in a separate PR.

Reply:
Thanks